src/python/pants/backend/python/subsystems/debugpy.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import annotations
+
+from pants.backend.python.goals import lockfile
+from pants.backend.python.goals.lockfile import GeneratePythonLockfile
+from pants.backend.python.subsystems.python_tool_base import PythonToolBase
+from pants.backend.python.subsystems.setup import PythonSetup
+from pants.backend.python.target_types import EntryPoint
+from pants.core.goals.generate_lockfiles import GenerateToolLockfileSentinel
+from pants.engine.rules import collect_rules, rule
+from pants.engine.unions import UnionRule
+from pants.option.option_types import IntOption, StrOption
+from pants.util.docutil import git_url
+
+
+class DebugPy(PythonToolBase):
+    options_scope = "debugpy"
+    help = "An implementation of the Debug Adapter Protocol for Python (https://github.com/microsoft/debugpy)."
```

Comment:
The spelling of `adapter` here got me googling, and I found this: https://writingexplained.org/adapter-vs-adaptor-difference

The take-away from that read is that it doesn't really matter, but I think we want to be consistent here, and stick with only one way of spelling adapter/adaptor.